main.go

```diff
@@ -308,7 +309,7 @@ func Main() error {
 		},
 	}
 
-	fs = flag.NewFlagSet("oracall", flag.ContinueOnError)
+	//fs = flag.NewFlagSet("oracall", flag.ContinueOnError)
```

Comment:
Why?

Reply:
Thought this is deprecated. Had no idea on how and why this should be used.
 Generated an error when testing it 
oracall oracall -v -connect "rator/rator@zh4labratordb2.sns.sec:1521/RATOR"       
                                                               
15:36:11.275 ERR "ERROR" attrs={"error":"terminal command (oracall) doesn't define an Exec function"}

Reply:
Each ffcli.Command needs a new FlagSet.
Invoke as

    oracall call -v -connect ...